migration/src/test/java/org/axonframework/migration/AxonJakartaTest.java

```diff
@@ -0,0 +1,79 @@
+package org.axonframework.migration;
+
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+import static org.openrewrite.java.Assertions.mavenProject;
+import static org.openrewrite.maven.Assertions.pomXml;
+
+public class AxonJakartaTest implements RewriteTest {
```

Comment:
I think we can make this file package private.
```suggestion
class AxonJakartaTest implements RewriteTest {
```